Romy-about

```diff
@@ -0,0 +1,14 @@
+layout: page
+title: RomyS
+
+---I am Romy's About Page
+
+## What I do
+I try to make complex stuff easy to understand!
+
+## My favorite things
+Travel around the world
+Read 
+Hike
+
```

Comment:
Are you sure you don't want to add - [img](https://thumb7.shutterstock.com/display_pic_with_logo/76219/373605241/stock-photo-beautiful-brunette-taking-a-nap-lying-on-the-couch-at-home-373605241.jpg)